views/user_sessions/new.haml

```diff
@@ -0,0 +1,9 @@
+.login-form
+  %form.form-signin{action: '/user_sessions/create', method: 'POST'}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace